index.html

```diff
@@ -0,0 +1,63 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <header class="header">
+        <a href="index.html" class="logo">
+            My Logo
+        </a>
+        <nav class="icon-menu"><img src="images/icon-menu.png" alt="icon-menu" id="img-1"></nav>
+    </header>
+    <main class="main-content">
+        <section class="section-1">
```

Comment:
- [ ] Good work adding your background image inside your `section`. Kindly fix the position of it to match the Figma design

![image](https://github.com/2004-okari/MY-PORTFOLIO/assets/93630700/2d9b884e-aade-4dfd-8c58-84e0d2c39452)